app/controllers/DefermentController.scala

```diff
@@ -78,6 +78,19 @@ class DefermentController @Inject()(identify: IdentifierAction,
     )
   }
 
+  private[controllers] def redirectToSplitPayment(userAnswers: UserAnswers): Result = {
+    userAnswers.get(UserTypePage).getOrElse("No user type") match {
```

Comment:
use match statement with guard as the _ case will occur in the "No user type" scenario.